torchrl/envs/transforms/transforms.py

```diff
@@ -539,9 +516,22 @@ def __setattr__(self, key, value):
         if isinstance(value, Transform):
             value.set_parent(self)
         if isinstance(propobj, property):
-            if propobj.fset is None:
+            ancestors = list(__class__.__mro__)[::-1]
+            while isinstance(propobj, property):
+                if propobj.fset is not None:
+                    return propobj.fset(self, value)
+                propobj = getattr(ancestors.pop(), key, None)
+            else:
                 raise AttributeError(f"can't set attribute {key}")
-            return propobj.fset(self, value)
+
+        # if isinstance(propobj, property):
```

Comment:
do we want to keep this commented block?

Reply:
No, that should go away.